x-pack/plugins/monitoring/public/application/pages/no_data.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+// @ts-ignore
+import { NoData as NoDataComponent } from '../../components/no_data';
```

Comment:
Let me know your thoughts on this naming. We could potentially call this `NoDataPage` as an alternative, or rename the NoData component but I figured we should avoid trying to alter the angular UI.

Reply:
I prefer calling it `NoDataPage` as it's how I would name it if I was implementing it from scratch. I don't think we alter the angular UI if we call it `NoDataPage` since in Angular this is not a component it's a route definition.